deploy/cloudformation/static-host-pipeline.yml

```diff
@@ -358,6 +413,13 @@ Resources:
           - Name: DEST_BUCKET
             Value:
               Fn::ImportValue: !Join [':', [!Ref ProdStackName, 'BucketName']]
+          - Name: OAUTH_TOKEN
+            Type: PARAMETER_STORE
+            Value: /esu/github/ndlib-git/oauth
+          - Name: API_URL
+            Value: !Ref ApiURL
```

Comment:
As an example, here you could just do:
`!Sub "https://api.github.com/repos/${SourceRepoOwner}/${SourceRepoName}/statuses`
and that should evaluate to:
`"https://api.github.com/repos/ndlib/mellon-image-viewer/statuses".`